crates/ty/src/args.rs

```diff
@@ -77,6 +77,14 @@ pub(crate) struct CheckCommand {
     pub(crate) extra_search_path: Option<Vec<SystemPathBuf>>,
 
     /// Python version to assume when resolving types.
+    ///
+    /// The Python version affects the allowed syntax and types of the standard library.
+    ///
+    /// By default, the Python version is inferred as the lower bound of the project's
+    /// `requires-python` field from the `pyproject.toml`, if available. Otherwise, the latest
+    /// version supported by ty is used, which is currently 3.13.
+    ///
+    /// ty will not infer the Python version from the Python environment at this time.
```

Comment:
I'd like to add this, but it won't make the release tomorrow and this is notable.